ampproject/amphtml
# Revert "amp-list: Fix Bind.rescan vs. diffing race condition"

Reverts ampproject/amphtml#32650
Fixes `b/182456351`. Breaks `b/152928375#comment48`.

@choumx: lmk if you prefer to fix a different way.

Would also love to chat with you about any ideas you may have for a a path to better maintainability for amp-list's email-specific features.

**to test**
- Use charles proxy to verify that current master breaks docs comment email reply button.
- Then also verify the bug is fixed with this branch
